code/src/components/FirstQuestion.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+const FirstQuestion = ({username, handleInputChange, backSlide, forwardSlide}) => {
+    return (
+        <div className='container'>
+            <div className='form-container'>
+                <label htmlFor='username'>Please enter your name</label>
+                <input  
+                value={username} 
+                id={username} 
+                type="text" 
+                onChange={handleInputChange}
+                />
```

Comment:
I'm not entirely sure about this, but I think there should be a  `<form>` element around the input for better accessibility